components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/authz/validators/AbstractResponseTypeRequestValidator.java

```diff
@@ -187,6 +193,44 @@ public OAuth2ClientValidationResponseDTO validateClientInfo(HttpServletRequest r
         }
     }
 
+    private void isHybridFlowRequest(HttpServletRequest request, OAuthAppDO appDO)
+            throws InvalidOAuthClientException {
+        String responseType = request.getParameter(OAuthConstants.OAuth20Params.RESPONSE_TYPE);
+        boolean hybridFlowEnabled = appDO.isHybridFlowEnabled();
+
+        if (OAuth2Util.isHybridResponseType(responseType)) {
+            if (!hybridFlowEnabled) {
+                if (log.isDebugEnabled()) {
+                    log.debug("Hybrid flow is not enabled for the application with client ID: "
+                            + appDO.getOauthConsumerKey());
+                }
+                throw new InvalidOAuthClientException("Hybrid flow is not enabled for the application.");
+            }
+
+            String configuredHybridFlowResponseType = appDO.getHybridFlowResponseType();
+            if (!isRequestedResponseTypeConfigured(responseType, configuredHybridFlowResponseType)) {
+                if (log.isDebugEnabled()) {
+                    log.debug("Requested response type " + responseType + " is not configured for the hybrid flow " +
+                            "for the application with client ID: " + appDO.getOauthConsumerKey());
+                }
+
+                throw new InvalidOAuthClientException("Requested response type " + responseType +
+                        " is not configured for the hybrid flow for the application.");
+            }
+        }
+    }
+
+    private boolean isRequestedResponseTypeConfigured(String responseType, String configuredHybridFlowResponseType) {
+        Set<String> configuredResponseTypes = new HashSet<>(Arrays.asList(configuredHybridFlowResponseType.split(" ")));
```

Comment:
Add a new line after the method definition.

Reply:
Fixed with: https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2461/commits/fa18f868b15608d8379b69af197b41e73fa39e7c